### PR TITLE
retire V1 server schema from web api

### PR DESCRIPTION
**Pull request by alex-kovoy**
this PR partially addresses #1280 by removing this dependency from the client side.

### CODE REVIEW

File: lib/web/ui/server.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package ui
+
+import (
+	"sort"
+
+	"github.com/gravitational/teleport/lib/services"
+)
+
+// LabelDTO describes a label
```
Comment (klizhentas) [inline lib/web/ui/server.go]: DTO is not clear to me, also it's redundant suffix here, not necessary.

File: lib/web/ui/server.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	// Value is this label value
+	Value string `json:"value"`
+}
+
+// ServerDTO describes a server
+type ServerDTO struct {
+	// Name is this server name
+	Name string `json:"id"`
+	// ClusterName is this server cluster name
+	ClusterName string `json:"siteId"`
```
Comment (klizhentas) [inline lib/web/ui/server.go]: should be `clusterName`, I think you should rename this in the UI as we are deprecating `site` everywhere

Comment (alex-kovoy) [reply lib/web/ui/server.go]: I agree but I would like to do it separately from this change

File: lib/web/ui/server.go (hunk 1 above)
Comment (alex-kovoy) [reply lib/web/ui/server.go]: will change

**Comment by alex-kovoy** [comment]
retest this please

**Comment by russjones** [comment]
@gravitational-jenkins retest this please

**Review by klizhentas — APPROVED** [review]

File: lib/web/ui/server.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
+package ui
```
Comment (kontsevoy) [inline lib/web/ui/server.go]: License header is missing.

**Review by kontsevoy — CHANGES_REQUESTED** [review]

**Comment by alex-kovoy** [comment]
retest this please

**Pull request merged**